content/en/user-guide/web-application/stack-insights/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Stack Insights"
+tags: ["LocalStack Pro", "Stack Insights"] 
+weight: 3
+description: >
+  Stack Insights enable users to report AWS API usage telemetry of LocalStack runs to their LocalStack account.
+---
+
+Stack Insights enable LocalStack users to report AWS API usage telemetry of LocalStack runs to their LocalStack account. You can see which APIs are being used, which clients of integrations use particular services and API operations, which services cause the most API errors, and much more using Stack Insights. The Stacks widget and detailed Stack information is part of our Pro plan, while the graphical representation currently falls under our Team preview.
+
+## Getting started
+
+To get started with this feature, log in to your LocalStack account and start a LocalStack instance on your local machine. The LocalStack Dashboard will show the Stacks widget, which holds most of the vital information of recent and currently running stacks.
+
+The Stack widget will also display the number of API calls, services employed and the runtime duration for each spin-up. All the spin-ups are timestamped; hence you can easily navigate the stacks and check the number of API calls that succeeded and failed.
+
+## Detailed Stack information
```

Comment:
NIT: it would be really nice to have screenshots here, so people can understand better, what this means. 
As a non-pro user I would also be interested what feature I would get here if I was pro. 

But maybe screenshots are also problematic, e.g. they should to be updated if anything in the UI changes

Reply:
For now, I have kept the screenshots :)